src/gate/printer/garphml.cpp

```diff
@@ -0,0 +1,64 @@
+#include "graphml.h"
+
+std::string link_deskriptor(Link link)
+{
```

Comment:
The bracket should be at the same line with the correspondent function name. Here, and everywhere.